examples/server/utils.hpp

```diff
@@ -542,3 +553,43 @@ static json format_detokenized_response(const std::string & content) {
         {"content", content}
     };
 }
+
+static json format_error_response(const std::string & message, const enum error_type type) {
+    std::string type_str;
```

Comment:
If a switch is applied to an enum, it will expect the switch to have all enum values. So having a default value here is not necessary.

For example, if we delete one case in the switch statement:

```
examples/server/utils.hpp: In function ‘json format_error_response(const string&, error_type)’:
examples/server/utils.hpp:560:12: warning: enumeration value ‘ERROR_TYPE_UNAVAILABLE’ not handled in switch [-Wswitch]
  560 |     switch (type) {
```

Only `int code` is initialized because the compiler is not that intelligent.

Reply:
if you enable warning yes, up to you.